src/Collectors/DBQueryCollector.php

```diff
@@ -61,4 +57,23 @@ public static function getName(): string
     {
         return 'query-collector';
     }
+
+    private function getQueryName(string $sql): string
+    {
+        $fallback = 'Eloquent Query';
+
+        try {
+            $query_type = QuerySplitter::getQueryType($sql);
+            $tables = QuerySplitter::splitTables($sql);
```

Comment:
Interesting package, lots of regexes. Builds are failing, but it seems to do what we need it to do.

Reply:
Yeap, but it does the job, and it's also really fast.